<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MesonRewriterPage</class>
 <widget class="QWidget" name="MesonRewriterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meson project settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="c_tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="t_project">
      <attribute name="title">
       <string>Project</string>
      </attribute>
      <attribute name="toolTip">
       <string>Project settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="l_dispProject">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="l_project">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h3&gt;projectName&lt;/h3&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="MesonRewriterInputString" name="w_version" native="true">
         <property name="name" stdset="0">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MesonRewriterInputString" name="w_license" native="true">
         <property name="name" stdset="0">
          <string>License</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MesonRewriterInputString" name="w_mesonVersion" native="true">
         <property name="name" stdset="0">
          <string>Meson version</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string>+</string>
             </property>
             <property name="icon">
              <iconset theme="list-add">
               <normaloff>../../../../../.designer/backup</normaloff>../../../../../.designer/backup</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_2">
             <property name="text">
              <string>-</string>
             </property>
             <property name="icon">
              <iconset theme="list-remove">
               <normaloff>../../../../../.designer/backup</normaloff>../../../../../.designer/backup</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="l_status">
       <property name="text">
        <string>Status message...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_changed">
       <property name="text">
        <string>Num changed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MesonRewriterInputString</class>
   <extends>QWidget</extends>
   <header>settings/mesonrewriterinput.h</header>
   <container>1</container>
   <slots>
    <signal>configChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>emitChanged()</slot>
 </slots>
</ui>
